### ISSUE TITLE
Voteythumbs

### ISSUE THREAD

**Issue by gricey432** [UQComputingSociety/uqcs-hubot#40]
If a user in a channel (a small channel or a private chat) says a message containing the keyword 'voteythumbs' the bot should monitor that message and announce the vote to be positive if more than half the channel gives a thumbs up reaction to that message, and announce it to be negative if it gets more than half thumbs down.
You can subscribe to reactions on a message using the realtime message API, I'm not sure if this is already being used in the bot.

**Comment by lewisjb** [comment]
When would the bot declare if the vote was 'positive' or 'negative'?

**Comment by gricey432** [comment]
It should just say something in chat like "Voteythumbs started 10 minutes ago was successful, 5 up and 2 down votes"
This should happen as soon as the either up or down votes equals or passes 50% of the channel population

**Comment by lewisjb** [comment]
Ahh yes, sorry I misread and didn't see the 'thumbs down'

**Comment by gricey432** [comment]
The inspiration for this is that amongst the exec we've been voting on decisions in slack by going
`@channel voteythumbs on doing [something]`
Then we all react with up or down thumbs to vote yes or no to a decision

**Comment by JosephGarrone** [comment]
NO PLEASE! Don't close it

**Comment by gricey432** [comment]
@lewisjb close with message if there's a good reason this should never be implemented

**Comment by JosephGarrone** [comment]
Why would you reopen this?

**Comment by lewisjb** [comment]
@gricey432 I closed it because I implemented it. In retrospect I should have referenced the commit.
527075e130b27c28cffea88cb0aaa3b797f0a96

Leave the issue open for now as there are some changes that needs to be made:
1. Replacing the "vote started" message with uqcsbot reacting to the original message.
2. Removing bots (mainly uqcsbot) from the member count (to determine the min votes to declare a winner)
